resources/content/schema/admin/admin-auth.json

```diff
@@ -1,46 +1,70 @@
 {
     "authorize" : {
-        "[a-zA-Z]*" : "crud",
-        "[a-zA-Z]*\\..*" : "r",
-
-        ".*\\..*Id" : "cru",
-        ".*\\.data" : "cru",
+    	".*" : "",
+    	
+    	".*\\..*Id" : "r",
```

Comment:
This also causes issue with all the id fields in processInstance in user-auth.json.
The spreadsheet indicates those should not be readable, but because of this wildcard (in user-auth.json) you are accidentally exposing resourceId, runningProcessServerId, startProcessServerId bc you arent explicitly squashing them right?

...Just noting places where this wildcard might burn us.